starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +22,45 @@ var words = [
   'crackpot'
 ];
 
+function findTheLongestWord(words){
+
+  for(let i = 0; i < words.length; i++ ){
+    if(words[i].length > longestWord.length)
+    longestWord = words[i];
+  }
+  console.log("The longest word is " + longestWord);
+}
+
+findTheLongestWord(arrayOfWords);
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+let counter = 0;
+
+function sumArray(nums){
+  for(let i = 0; i < nums.length; i++)
+    counter += nums[i];
+    console.log(counter);
+}
```

Comment:
I see that you didn't return something with some other functions as well. I'd recommend to return something with most functions. Though not all functions necessarily should return something, it's good practice to do for the most part, especially when you have a function so specific as sumArray() where you'd return the sum